### PR TITLE
[RHEL-9] Use recommended format for JIRA links

### PR DESCRIPTION
**Pull request by jkonecny12**
Use recommended format for JIRA links and improve output logs to make them less confusing.

### CODE REVIEW

**Review by jstodola — APPROVED** [review]
The changes look good to me. What I've just realized is that the guidelines in docs/commit-log.rst should also be updated to use the new Jira format.

**Comment by jkonecny12** [comment]
Good point. Changed and also creating a new PR for removing outdated scripts.

File: docs/commit-log.rst [hunk 1]
```diff
@@ -5,22 +5,13 @@ git commit messages for anaconda should follow a consistent format.  The
...
-   line to indicate the bugzilla.redhat.com bug number addressed in this
-   patch.  The bug number is optional since there may be no bug number,
-   but if you have one you are addressing, please include it on the
-   summary line.  Lastly, the summary lines need to be short.  Ideally
-   less than 75 characters, but certainly not longer than 80.
+   change in the patch.  The summary lines need to be short.
+   Ideally less than 75 characters, but certainly not longer than 80.
 
-   Here are acceptable first lines for git commit messages::
+   Here are acceptable first lines for git commit message::
```
Comment (jstodola) [inline docs/commit-log.rst:11]: Either add more examples or use singular.

File: docs/commit-log.rst [hunk 2]
```diff
@@ -30,34 +21,25 @@ following are rules to follow when committing a change to the git repo:
...
-   follow one of two formats for specifying the bug number in a RHEL commit.
-
-   a)  Put the bug number on the summary line in (#BUGNUMBER) format.  Bugs
-       listed this way are treated as 'Resolves' patches in the RHEL
-       universe.
-
-   b)  If you have a patch that is Related to or Conflicts with another bug,
-       you may add those lines to the end of the long commit message in this
-       format::
+3) For RHEL bugs, all commits need to reference a bug issue name.
```
Comment (jstodola) [inline docs/commit-log.rst:24]: You can mention that both RHEL and CentOS Stream contributions need a bug ID.
Also, point the users to https://issues.redhat.com/projects/RHEL/issues as a place where to report such bugs.

File: docs/commit-log.rst (hunk 1 above)
Comment (jkonecny12) [reply docs/commit-log.rst]: Fixed.

File: docs/commit-log.rst (hunk 2 above)
Comment (jkonecny12) [reply docs/commit-log.rst]: Fixed.

**Pull request merged**